content/en/blog/_posts/2018-05-29-announcing-kustomize.md

```diff
@@ -8,12 +8,11 @@ date: 2018-05-29
 
 [**kustomize**]: https://github.com/kubernetes-sigs/kustomize
 [hello world]: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/helloWorld
-[kustomization]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization
 [mailing list]: https://groups.google.com/forum/#!forum/kustomize
 [open an issue]: https://github.com/kubernetes-sigs/kustomize/issues/new
-[subproject]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/0008-kustomize.md
+[subproject]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/2377-Kustomize/README.md
```

Comment:
This [commit](https://github.com/kubernetes/enhancements/commit/3e4e4171929e03de851d3fe3946f7d1e5d10bf33) shows that the page migrated from `~/0008-kustomize.md` to `~/2377-Kustomize/README.md`.

![image](https://github.com/kubernetes/website/assets/73531614/fe748775-f49d-4ffc-b6df-ffa5f3966891)